ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/utils/VicVmComparator.java

```diff
@@ -51,20 +53,69 @@ public VicVmComparator(
 	 */
 	@Override
 	public int compare(String a, String b) {
-		try {
-		    ModelObject mo_A = (ModelObject) base.get(a);
-	        ModelObject mo_B = (ModelObject) base.get(b);
-			int result = 0;
-			result = getStringPropertyFromVm(mo_A)
-				.compareTo(getStringPropertyFromVm(mo_B));
-			if (result == 0) {
-			    result = mo_A.hashCode() - mo_B.hashCode();
-			}
+	    ModelObject modObjA = (ModelObject) base.get(a);
+        ModelObject modObjB = (ModelObject) base.get(b);
```

Comment:
Indentation seems off here. Not sure if it's editor-related or due to Github diff rendering.